js/Common.js

```diff
@@ -538,7 +538,23 @@ if (!CRM.vars) CRM.vars = {};
     });
   }
 
-  function getStaticOptionMarkup(staticItems) {
+  function renderQuickAddMarkup(quickAddLinks) {
+    if (!quickAddLinks || !quickAddLinks.length) {
+      return '';
+    }
+    let markup = '<div class="crm-entityref-links crm-entityref-quick-add">';
+    CRM.config.quickAdd.forEach((link) => {
+      if (quickAddLinks.includes(link.path)) {
```

Comment:
```suggestion
      if (_.includes(quickAddLinks, link.path)) {
```

Reply:
I've been trying to use more vanilla and less lo-dash recently, as that seems to be the direction the whole internet is headed. Did you find a bug here or is it just a style thing?

Reply:
It's a bug.

Reply:
Ah, I think I found the cause - quickAddLinks is a string but was supposed to be an array. One sec...

Reply:
That's why I made the Line 580 suggestion.